domain/entity/ActionSpace.ts

```diff
@@ -4,23 +4,32 @@ import { Game } from "./Game";
 import { Resources } from "./Resources";
 
 export class ActionSpace {
-    constructor(id: ActionSpaceId, action: Action, replenishment?: Replenishment, dwarf?: Dwarf) {
+    constructor(
+        id: ActionSpaceId,
+        action: Action,
+        replenishment?: Replenishment,
+        dwarf?: Dwarf,
+        newBornDwarf?: Dwarf
```

Comment:
So we'll be using this to know that this dwarf should only be fed 1 PN?

(Or maybe we'll think about that later when we'll actually need this? ^^)

Reply:
Hmmm I'm rather for the option "Or maybe we'll think about that later when we'll actually need this?" for now 😬 
Having sth like `isBaby` in dwarf might be more clean 🤔